packages/xo-web/src/xo-app/hub/resource-form.js

```diff
@@ -52,9 +79,52 @@ export default decorate([
           onChange={effects.onChangePool}
           predicate={poolPredicate}
           required
-          value={state.pools}
+          value={pools}
         />
       </FormGrid.Row>
+      {install && isMultiplePools ? (
```

Comment:
If it's getting too complex with too many conditions for whether you want to show a select or not, it might mean that you need to have 2 separate modals for the 2 actions.